domains/risshi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "codeRisshi25",
+        "email": "risshirajsen@gmail.com"
+    },
+    "record": {
+        "CNAME": "www.coderisshi25.github.io"
```

Comment:
404 CNAME record. Please have content.

Reply:
made changes to the CNAME record , please review